sample-repo/example/ExampleTest.scala

```diff
@@ -0,0 +1,8 @@
+import dep.Dep
+
+object ExampleTest {
+  def test(): Unit = {
```

Comment:
To keep this consistent, this object should have a main method, since this is required by main_class